symengine/lib/symengine_wrapper.pyx

```diff
@@ -16,6 +16,10 @@ import warnings
 
 include "config.pxi"
 
+from setup import use_cysignals
+if use_cysignals:
+    include "cysignals/signals.pxi"
```

Comment:
I actually wanted to know whether there is a workaround for this?
I'm not sure this would work.